spec/models/file_set/file_set_derivatives_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:file)        { build(:department_file, id: '1234') }
   let(:derivatives) { CurationConcerns::DerivativePath.derivatives_for_reference(file) }
 
-  before { LakeshoreTesting.reset_derivatives }
+  before(:each) { LakeshoreTesting.reset_derivatives }
```

Comment:
I don't think before is the same as before(:each), because the test fails without it because the files are definitely still there from the previous test, with seed 14171 (which runs them in the order in which the persisting files are detected). Adding the :each actually clears the files and the test passes.